packages/web-react/src/components/UNSTABLE_Avatar/README.md

```diff
@@ -0,0 +1,114 @@
+# UNSTABLE Avatar
+
+⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+Please use it with caution.
+
+The `UNSTABLE_Avatar` component is used to display a user's profile picture or initials.
+
+```jsx
+import { UNSTABLE_Avatar } from '@lmc-eu/spirit-web-react';
+
+<UNSTABLE_Avatar aria-label="Profile of Jiří Bárta">
+  <span aria-hidden="true">JB</span>
+</UNSTABLE_Avatar>;
+```
+
+## Square
+
+Add `isSquare` prop to make the avatar a square.
```

Comment:
This could be set like `shape=[ circle (default) | square ]`.

I can see there is probably an analogy with the `Button` component and its `isSquare` prop, although this is not quite the same situation…

I'm OK with both `square` and `isSquare` variants. Just if you decide for the `square` enum prop, CSS should be changed too.

https://github.com/lmc-eu/spirit-design-system/pull/1469/files#r1647648629

Reply:
I'll leave it as it is for now, it is an unstable component, so we can change it later if we decide to.